xml/schema/types/transits-2-9-6.xsd

```diff
@@ -44,7 +44,8 @@
           <xs:complexType>
             <xs:sequence>
               <xs:element name="systemName" type="systemNameType" minOccurs="0" maxOccurs="1"/>
-              <xs:element name="userName" type="userNameType" minOccurs="0" maxOccurs="1"/>
+	      <xs:element name="userName" type="userNameType" minOccurs="0" maxOccurs="1"/>
+	      <xs:element name="comment" type="userNameType" minOccurs="0" maxOccurs="1"/>
```

Comment:
Bother, I copied it from section. There is actually a "commentType" defined in general.xsd, which seems the most popular. Do I update my fork/branch and re-request a pull or what? Im a bit fuzzy about this process.